lib/internal/crypto/keys.js

```diff
@@ -160,6 +161,10 @@ const [
   }
 
   class AsymmetricKeyObject extends KeyObject {
+    // eslint-disable-next-line no-useless-constructor
+    constructor(type, handle) {
+      super(type, handle);
+    }
```

Comment:
Theres should be an empty line between class members of `AsymmetricKeyObject`.

Reply:
```suggestion
    }

```